test/testLuaTemplater.cpp

```diff
@@ -58,40 +58,6 @@ TEST(testLuaTemplater, basicsubstitution) {
     string result2 = mytemplate.render(source);
     EXPECT_EQ(expectedResult, result2);
 }
-//TEST(testLuaTemplater, basicsubstitution_from_vec) {
```

Comment:
> remove commented code that did not compile

are we sure this isnt testing something important, and we shouldnt just fix it?  (I havent checked this point; not saying it is/isnt; just posing the question)